lighthouse-core/scripts/gcp-collection/extract-data.sh

```diff
@@ -3,21 +3,32 @@
 set -euxo pipefail
 
 ##
-# @license Copyright 2020 The Lighthouse Authors. All Rights Reserved.
+# @license Copyright 2021 The Lighthouse Authors. All Rights Reserved.
```

Comment:
we don't update copyright years for renames, the year indicates the earliest point at which this code existed